### PR TITLE
Refactor add frontend and Setup CI

### PR DESCRIPTION
**Pull request by mkZH0740**

### CODE REVIEW

File: requirements.txt
Comment (SoyMark) [inline requirements.txt:1]: Not sure what is necessary after code refactoring, but maybe we should at least keep Django here.

Comment (mkZH0740) [reply requirements.txt:1]: Django is now inside backend folder

Comment (SoyMark) [reply requirements.txt:1]: Oh I see

**Review by SoyMark — APPROVED** [review]
Work perfectly. Great milestone. Thanks for the contribution.

**Pull request merged**